src/main/java/core/basesyntax/service/operation/ReturnOperation.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.operation;
+
+public class ReturnOperation implements OperationHandler {
```

Comment:
The class `ReturnOperation` should be placed inside the `strategy` package according to the checklist.